cypress/e2e/opening-hours-editor.cy.ts

```diff
@@ -140,50 +142,90 @@ const checkConfirmationDialog = ({
   end,
   endDate,
 }: Required<OpeningHourFormType>) => {
-  cy.getBySel("cy-opening-hours-editor-confirm-add-repeated-form")
+  cy.getBySel("opening-hours-editor-confirm-add-repeated-form")
     .should("be.visible")
     .and("contain", openingHourCategory)
     .and("contain", start)
     .and("contain", end)
     .and("contain", reverseDateString(endDate));
 };
 
-const confirmRepeatedOpeningHourForm = () => {
-  cy.getBySel("cy-opening-hours-editor-form__confirm").click();
+const confirmAddRepeatedOpeningHourForm = () => {
+  cy.getBySel("opening-hours-editor-form__confirm").click();
 };
 
-const checkOpeningHoursExistAdmin = ({
+const validateOpeningHoursPage = ({
   openingHourCategory,
   start,
   end,
 }: OpeningHourFormType) => {
-  return cy
-    .getBySel("opening-hours-editor-event-content")
+  cy.getBySel("opening-hours-week-list")
     .should("be.visible")
     .and("contain", openingHourCategory)
     .and("contain", `${start} - ${end}`);
 };
 
-const checkOpeningHoursExistPage = ({
+const validateAtLeastOneOpeningHoursExistAdmin = ({
   openingHourCategory,
   start,
   end,
 }: OpeningHourFormType) => {
-  cy.getBySel("opening-hours-week-list")
+  return cy
+    .get('tbody[role="presentation"]')
     .should("be.visible")
-    .and("contain", openingHourCategory)
-    .and("contain", `${start} - ${end}`);
+    .find('div[data-cy="opening-hours-editor-event-content"]')
+    .should("have.length.gte", 1)
+    .each((element) => {
+      cy.wrap(element)
+        .should("contain", openingHourCategory)
+        .and("contain", `${start} - ${end}`);
+    });
 };
 
-const checkMultipleOpeningHoursExistAdmin = () => {
-  cy.get('tbody[role="presentation"]')
+const validateOpeningHoursChangesAdmin = ({
+  openingHourCategory,
+  originalStart,
+  originalEnd,
+  updatedStart,
+  updatedEnd,
+  editSeriesFromIndex,
```

Comment:
In general I find it very hard to grok what the parameter `editSeriesFromIndex` actually means. Perhaps it is just me.

Based on careful reading of the test consider refactoring the method to just check that you have the expected number of opening hour entries with the provided start and end times. Then the test can call this function twice and check that we have the expected number of original times and the expected number of updated times.